modules/op-release-notes-1-14.adoc

```diff
@@ -362,3 +362,13 @@ With this update, {pipelines-title} General Availability (GA) 1.14.5 is availabl
 * Before this update, when you used the web console and clicked a pipeline in the overview page, the pipeline details page did not contain information about tasks in the pipeline. With this update, when you click a pipeline in the overview page, the pipeline details page contains the required information.
 
 * Before this update, when you configured {tekton-chains} to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and {tekton-chains} attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, {tekton-chains} does not attempt to store OCI artifacts.
+
+[id="release-notes-1-14-6_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.6
+
+With this update, {pipelines-title} General Availability (GA) 1.14.5 is available on {OCP} 4.12 and later versions.
```

Comment:
Should this be `1.14.6`? If this is intentional, this probably needs some explaining because the version mismatch looks like a typo.

Reply:
It was a typo. Thanks, great catch